azurerm/internal/services/securitycenter/resource_arm_security_center_subscription_pricing.go

```diff
@@ -44,6 +52,21 @@ func resourceArmSecurityCenterSubscriptionPricing() *schema.Resource {
 					string(security.Standard),
 				}, false),
 			},
+			"resource_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "VirtualMachines",
+				ValidateFunc: validation.StringInSlice([]string{
+					"AppServices",
+					"ContainerRegistry",
```

Comment:
given the others are pluralised - just double-checking this isn't meant to be `ContainerRegistries`?

Reply:
The list came from the `List` api: https://github.com/terraform-providers/terraform-provider-azurerm/pull/8549#discussion_r498363634
but good spot, we'll need to keep an eye out for this being "brought in line".

Reply:
Yes, consistency is a problem, it is indeed singular in this case.